newrelic-api/src/main/java/com/newrelic/api/agent/HttpParameters.java

```diff
@@ -42,13 +46,16 @@
     private final ExtendedInboundHeaders extendedInboundResponseHeaders;
 
     protected HttpParameters(String library, URI uri, String procedure, InboundHeaders inboundHeaders) {
-        this(library, uri, procedure, inboundHeaders, null);
+        this(library, uri, procedure, null, null, inboundHeaders, null);
     }
 
-    protected HttpParameters(String library, URI uri, String procedure, InboundHeaders inboundHeaders, ExtendedInboundHeaders extendedInboundHeaders) {
+    protected HttpParameters(String library, URI uri, String procedure, Integer statusCode, String statusText, InboundHeaders inboundHeaders,
+            ExtendedInboundHeaders extendedInboundHeaders) {
```

Comment:
This constructor was changed and would break anyone subclassing HttpParameters and using it.
There is a simple workaround which is to create a constructor that receives a Builder.